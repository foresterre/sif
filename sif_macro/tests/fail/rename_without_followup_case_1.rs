#[macro_use]
extern crate sif_macro;

#[parameterized]
#[case(0)]
#[rename(after_case_on_nothing)]
pub(crate) fn my_test(a: u8) {}

fn main() {}
