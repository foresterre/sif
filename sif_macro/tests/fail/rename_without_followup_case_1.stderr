error: custom attribute panicked
 --> $DIR/rename_without_followup_case_1.rs:4:1
  |
4 | #[parameterized]
  | ^^^^^^^^^^^^^^^^
  |
  = help: message: [sif_macro] Found an unused 'rename' attribute for test 'my_test'; verify that you correctly put 'rename' attributes on top of 'case' attributes. On top, 'rename' attributes do not stack. Hints:
          At test fn 'my_test':
              Attribute 'after_case_on_nothing' [stack index: 1] used incorrectly

          (note: stack index starts at 0, after #[parameterized] attribute)
